components/EventsComponent.vue

```diff
@@ -18,17 +18,31 @@ const d = Object.fromEntries(Object.entries(data).filter(([k, v]) =>
 ).sort())
 </script>
 
+<style scoped>
+.eventsList article:first-of-type {
+  border-top-width: 0;
+}
+.eventsList article {
+  border-top: 1px solid var(--vp-c-divider);
+}
+.eventsList h3 {
+  margin-top: 18px;
+}
+.eventsList :deep(table p) {
+  margin: 0;
+}
+</style>
+
 <template>
- <template v-for="(v, k) in d">
-  <h3 :id="k" tabindex="-1">
-   <code>{{ k }}</code>
-   <a class="header-anchor" :href="'#' + k"></a>
-  </h3>
+ <section class="eventsList">
+  <article v-for="(v, k) in d">
+   <h3 id="k" tabindex="-1">
+    <code>{{ k }}</code>
+    <a class="header-anchor" :href="'#' + k"></a>
+   </h3>
 
-  <span v-if="v.text" v-html="v.text" />
+   <span v-if="v.text" v-html="v.text" />
```

Comment:
In practice this will contain `<p>` tags and as per https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-span-element and https://html.spec.whatwg.org/multipage/dom.html#phrasing-content `<span>` tags must not include `<p>` tags.

So make this a `<div>`.